frontend/src/components/cart/Payment.js

```diff
@@ -1,219 +1,213 @@
-    <h1 className="mb-4">Click Here:</h1>
-<button className="btn btn-block py-3" onClick={payNow}>Pay Now {` - ${(orderInfo && orderInfo.totalPrice)}`} RWF
-<input type="hidden" name="amount" value={orderInfo.totalPrice}></input>
-<input type="hidden" name="currency" value="RWF" ></input>
-<input type="hidden" name="comment" value="Order 122"></input>
-<input type="hidden" name="client_token" value=""></input>
-<input type="hidden" name="return_url" value="/success"></input>
-<input type="hidden" name="app_id" value="10c91e7ce9366b9641a7b999bf76ccb9"></input>
-<input type="hidden" name="app_secret" value="JDJ5JDEwJC4yaUpy"></input>
-<input type="image" src="images/paynw.png" alt="" onclick="document.afripayform.submit();"></input>
- </button>
- </form>
-            <div className="col-25 col-lg-13 mt-9 order-confirm">
-                    <form className="shadow-lg">
-        
-                        <h1 className="mb-4">Card Info</h1>
-                        <div className="form-group">
-                            <label htmlFor="card_num_field">Card Number</label>
-                            <CardNumberElement
-                                type="text"
-                                id="card_num_field"
-                                className="form-control"
-                                options={options}
-                            />
-                        </div>
-
-                        <div className="form-group">
-                            <label htmlFor="card_exp_field">Card Expiry</label>
-                            <CardExpiryElement
-                                type="text"
-                                id="card_exp_field"
-                                className="form-control"
-                                options={options}
-                            />
-                        </div>
-
-                        <div className="form-group">
-                            <label htmlFor="card_cvc_field">Card CVC</label>
-                            <CardCvcElement
-                                type="text"
-                                id="card_cvc_field"
-                                className="form-control"
-                                options={options}
-                            />
-                        </div>
-
-
-                        <button
-                            id="pay_btn"
-                            onClick={payWithCard}
-                            type="submit"
-                            className="btn btn-block py-3"
-                        >
-                            Pay $  {` - ${(orderInfo && orderInfo.totalPrices)}`}
-                        </button>
-                        
-                    </form>
-                </div>
-
-                </div>
-            
-
-
-
-        
-
-            </div>       
-</Fragment>
-    )
-}
-export default Payment
+  };
```

Comment:
this was previously your "afripayform" method which was never used. Added the payment submission step before order creation